procstats/delaystats_linux.go

```diff
@@ -0,0 +1,32 @@
+package procstats
+
+import (
+	"errors"
+	"syscall"
+
+	"github.com/segmentio/taskstats"
```

Comment:
What motivated to bring this in-house instead of taking a dependency to another repository?

Reply:
It's currently dependent on a PR that's waiting to be merged.  Once it's merged we can change this to be the vendor repository again.